docs/source/resources/scyllaclusters/storage-resizing.md

```diff
@@ -0,0 +1,99 @@
+# Resizing storage in ScyllaCluster
+
+Due to limitations of the underlying StatefulSet, ScyllaClusters don't allow storage changes. The following procedure describes how to adjust the storage manually.
+
+Let's use the following pods and StatefulSet in the resize examples below.
```

Comment:
It should be generic enough to explain it needs to be done for every rack (StatefulSet) that requires changes.